omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -20,9 +20,4 @@
   		    http://www.springframework.org/schema/util
   		    http://www.springframework.org/schema/util/spring-util-3.0.xsd">
 
-	<context:component-scan base-package="org.openmrs.module.tag.web.controller" />
-
-	<bean class="org.openmrs.ui.framework.StandardModuleUiConfiguration">
```

Comment:
These beans are invalid, since the module does not have uiframework dependencies. hence adding them will cause an error